src/Components/Buttons.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+class Buttons extends React.PureComponent {
+  render() {
+    return (
+      <div className="calculator-body">
+        <div className="screen">0</div>
+        <div className="keyboard">
+          <div className="row">
+            <button type="button" className="btn">AC</button>
+            <button type="button" className="btn">+/-</button>
+            <button type="button" className="btn">%</button>
+            <button type="button" className="btn">÷</button>
+          </div>
+          <div className="row">
+            <button type="button" className="btn">7</button>
+            <button type="button" className="btn">8</button>
+            <button type="button" className="btn">9</button>
+            <button type="button" className="btn">x</button>
+          </div>
+          <div className="row">
+            <button type="button" className="btn">4</button>
+            <button type="button" className="btn">5</button>
+            <button type="button" className="btn">6</button>
+            <button type="button" className="btn">-</button>
+          </div>
+          <div className="row">
+            <button type="button" className="btn">1</button>
+            <button type="button" className="btn">2</button>
+            <button type="button" className="btn">3</button>
+            <button type="button" className="btn">+</button>
+          </div>
+          <div className="row last-row">
+            <button type="button" className="btn">0</button>
+            <button type="button" className="btn">.</button>
+            <button type="button" className="btn">=</button>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
```

Comment:
I would recommend using a functional component instead of a class-based component. This is a core requirement for this milestone.